### ISSUE TITLE
Can't resolve correctly on IE11

### ISSUE THREAD

**Issue by xipeter** [superbiger/pinyin4js#10]
**Describe the bug**
This JS library works perfect except on IE11.

**To Reproduce**
1. Just run it on IE11.
2. Open the developer tools

**Expected behavior**
should work without any exception

**Screenshots**
![image](https://user-images.githubusercontent.com/32719912/55518851-6be07c80-562a-11e9-8e7a-9e794f9b57d5.png)

**Additional context**
Could you define these functions convertToPinyinString, convertToSimplifiedChinese, etc,. using ES5, otherwise it won't be supported by IE11.

**Comment by agavrilov** [comment]
We encountered the same problem. If you add ': function' between function name and parameters, it will make it compatible with IE11.

Original:
```javascript
getShortPinyin(str) {
        return PinyinHelper.PinyinHelper.getShortPinyin(str)
}
```
must be changed to:
```javascript
getShortPinyin: function(str) {
        return PinyinHelper.PinyinHelper.getShortPinyin(str)
}
```

**Comment by Kaifuny** [comment]
@agavrilov thanks.
@xipeter have fixed at 1.3.18
